### ISSUE TITLE
Avoid anonymous structs and enums

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/SourcetrailBugTracker#292]
There are anonymous struct nodes when using this syntax:
```
typedef struct {
   type_1 field1;
   type_2 field2;
   ....
} struct_name_t;
```

**Comment by egraether** [comment]
+1 by mail

**Comment by egraether** [comment]
Similar problem with `typedef union`

**Comment by mlangkabel** [comment]
why is this one labled as "bug"?

**Comment by egraether** [comment]
+1 by mail

**Comment by egraether** [comment]
Implemented with 2017.2.34
